### PR TITLE
expose a couple of specific sqlite3 error codes to Class object

### PR DESCRIPTION
**Pull request by TerrenceMcGuinness-NOAA**
**Description**

We request adding a couple of key sqlite3 error codes to be exposed by the wxflow SQLiteDB Class so the user can specifically query for the error strings with then the are appropriately encountered for a reaction.

**Type of change**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

The re-action to using try block when a uniqueness was violated was appropriately observed in the driver code using the class.

- [ ] pynorms
- [ ] pytests

**Checklist**

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

### CODE REVIEW

File: src/wxflow/sqlitedb.py [hunk 1]
```diff
@@ -17,6 +17,9 @@ class SQLiteDB:
 
     """
 
+    OperationalError = sqlite3.OperationalError
+    IntegrityError = sqlite3.IntegrityError
+
```
Comment (aerorahul) [inline src/wxflow/sqlitedb.py]: This works, but if you look at classes, having a:

```
class SQLiteDBError(sqlite3.Error):

    OperationalError = sqlite3.OperationalError
    IntegrityError = sqlite3.IntegrityError
```
would be the way to provide these extensions

**Review by aerorahul — APPROVED** [review]
looks good. 
will merge when the tests and norms pass.

File: src/wxflow/sqlitedb.py (hunk 1 above)
Comment (TerrenceMcGuinness-NOAA) [reply src/wxflow/sqlitedb.py]: Ok, updated

**Comment by TerrenceMcGuinness-NOAA** [comment]
updated pynorms for spaces

**Pull request merged**